catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form.temp.view.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Can we rename this to `search-form.collection.view.js`?

Reply:
That name exists already; renamed to be `search-form-tab-container.view.js`